src/installation/guides/chroot.md

```diff
@@ -0,0 +1,298 @@
+
+[nvi(1)](https://man.voidlinux.org/nvi.1) is available in the chroot, but you
+may wish to install your preferred text editor at this time.
+
+For glibc builds, generate locale files with
+
+```
+(chroot) # xbps-reconfigure -f glibc-locales
+```
+
+### Set a Root Password
+
+[Configure at least one super user account.](../../config/users-and-groups.md)
+Other user accounts can be configured later, but there should either be a root
+password, or a new user account with [sudo(8)](https://man.voidlinux.org/sudo.8)
+privileges.
+
+To set a root password, run
+
+```
+(chroot) # passwd
+```
+
+### Configure fstab
+
+Configure [fstab(5)](https://man.voidlinux.org/fstab.5). This file can be
+automatically generated from currently mounted filesystems by copying the file
+`/proc/mounts`.
+
+```
+(chroot) # cp /proc/mounts /etc/fstab
+```
+
+Remove lines in `/etc/fstab` that refer to `proc`, `sys`, `devtmpfs` and `pts`.
+
+Replace references to `/dev/sdXX`, `/dev/nvmeXnYpZ`, etc. with their respective
+UUID, which can be found by found by running
+[blkid](https://man.voidlinux.org/blkid.8). Referring to filesystems by their
+UUID guarantees they will be found even if they are assigned a different name at
+a later time. In some situations, such as booting from USB, this is absolutely
+essential. In other situations, such as SATA or NVME, drives will generally
+always have the same name unless drives are physically added or removed. This
+step may not be strictly necessary but is almost always recommended.
+
+Change the last zero of the entry for `/` to `1`, and the last zero of every
+other line to `2`. These values configure the behaviour of
+[fsck(8)](https://man.voidlinux.org/fsck.8).
+
+For example, the partition scheme used throughout previous examples yields the
+following fstab:
+
+```
+/dev/sda1       /boot/EFI   vfat    rw,relatime,[...]       0 0
+/dev/sda2       /           ext4    rw,relatime             0 0
+```
+
+The fstab is modified thus with information from `lsblk -f`:
+
+```
+UUID=6914[...]  /boot/EFI   vfat    rw,relatime,[...]       0 2
+UUID=dc1b[...]  /           ext4    rw,relatime             0 1
+```
+
+> Note: The output of `/proc/mounts` will have a single space between each
+> field. The columns are aligned here for readability.
+
+Add an entry to mount `/tmp` in ram:
+
+```
+tmpfs           /tmp        tmpfs   defaults,nosuid,nodev   0 0
+```
+
+If using swap space, add an entry for any swap partitions:
+
+```
+UUID=1cb4[...]  swap        swap    rw,noatime,discard      0 0
+```
+
+## Installing GRUB
+
+**On a BIOS computer**, install the package `grub`, then run `grub-install
+/dev/sdX` where `/dev/sdX` is the drive (not partition) that you wish to install
+GRUB to.
+
+```
+(chroot) # xbps-install grub
+(chroot) # grub-install /dev/sda
+```
+
+**On a UEFI computer**, install either `grub-x86_64-efi` or `grub-i386-efi`
+depending on your architecture then run `grub-install`, optionally specifying a
```

Comment:
", depending on your architecture, "

Reply:
Yes. What do you think about the comma after "On a UEFI computer" and "On a BIOS computer"? I'm starting to think they shouldn't be there

Reply:
I think those commas are necessary.